### ISSUE TITLE
Can't run e2e in CI

### ISSUE THREAD

**Issue by mariarodrigu** [webdriverio/webdriverio#2786]
## The problem
My E2E tests works fine locally but got the error "ERROR: connect ECONNREFUSED 127.0.0.1:4444 error" in CI/Docker

## Environment
* WebdriverIO version: 4.13.1
* Node.js version: 8.9.1
* Using Chromedriver without selenium
* Async run
* Additional wdio packages used:
    wdio-chromedriver-service: 0.1.3
    wdio-cucumber-framework: 2.1.0
    wdio-json-reporter: 0.3.1

## Details
I'm pretty sure I'm missing something... my tests run ok locally, but when I try to run them in CI/Docker Container, they fail with the error described above. I'm pasting my conf below in case I'm missing something.
```

exports.config = {
  specs: ['./tests/e2e/wdio/features/*.feature'],
  exclude: [],
  maxInstances: 1,
  capabilities: [{
    maxInstances: 1,
    browserName: 'chrome',
    chromeOptions: {
      args: [
        '--no-sandbox',
        '--window-size=1280,800',
        '--headless',
        '--disable-gpu',
      ] },
  }],
  sync: true,
  logLevel: 'verbose',
  coloredLogs: true,
  deprecationWarnings: true,
  bail: 1,
  screenshotPath: './errorShots/',
  baseUrl: 'http://localhost',
  waitforTimeout: 10000,
  connectionRetryTimeout: 90000,
  connectionRetryCount: 3,
  // path: '/',
  services: ['chromedriver'],
  chromeDriverArgs: ['--port=4444'],
  framework: 'cucumber',
  reporters: ['json'],
  reporterOptions: {
    outputDir: './tests/e2e/wdio/reports/',
  },
  cucumberOpts: {
    require: ['./tests/e2e/wdio/features/step_definitions/**.js'],
    backtrace: false,
    compiler: [],
    dryRun: false,
    failFast: false,
    format: ['pretty'],
    colors: true,
    snippets: true,
    source: true,
    profile: [],
    strict: false,
    tags: [],
    timeout: 20000,
    ignoreUndefinedDefinitions: false,
  },
  // before: () => {
  //   Object.keys(commands).forEach((key) => {
  //     browser.addCommand(key, commands[key]);
  //   });
};
```

Thanks in advance!

**Comment by emilyrohrbough** [comment]
If you wrap you config block in ``` tags it will display nicer

**Comment by mariarodrigu** [comment]
Done @emilyrohrbough , thanks for the tip :)

**Comment by BorisOsipov** [comment]
>but got the error "ERROR: connect ECONNREFUSED 127.0.0.1:4444 error" in CI/Docker

It means your selenium(actually chromedriver) service isn't available on 127.0.0.1:4444 and isn't related to wdio.
Check your CI\docker\etc setup.
Join [our Gitter](https://gitter.im/webdriverio/webdriverio) if you have more questions. Thanks!
